app/celer-dump-data.cc

```diff
@@ -702,6 +702,17 @@ void print_optical_material_data(ImportData::ImportOpticalMap const& iom)
         cout << POM_STREAM_VECTOR(mid, abs, absorption_length, IU::len);
     }
     cout << endl;
+    cout << "\n## WLS";
+    cout << header;
+    for (auto const& [mid, val] : iom)
+    {
+        auto const& wls = val.wls;
+        cout << POM_STREAM_SCALAR(mid, wls, mean_num_photons, IU::unitless);
+        cout << POM_STREAM_SCALAR(mid, wls, time_constant, IU::time);
+        cout << POM_STREAM_VECTOR(mid, wls, absorption_length, IU::len);
+        cout << POM_STREAM_VECTOR(mid, wls, component, IU::mev);
```

Comment:
I find that the WLS component is unitless (it is a distribution of relative population along the re-emission energy axis and is used to build an inverse CDF to sample the emitted photon energy) - sorry about a wrong guess during our discussion at the hackaton.

Reply:
Ohh I see. Thanks, fixing that now.